meshtastic/mesh_interface_runtime/node_view.py

```
@@ -0,0 +1,640 @@
+        meshtastic.node.Node
+            The Node object corresponding to nodeId.
+
+        Raises
+        ------
+        MeshInterfaceError
+            If channel retrieval repeatedly fails or times out.
+        """
+        MeshInterface = self._interface.__class__
+        if nodeId in (LOCAL_ADDR, BROADCAST_ADDR):
+            return self.localNode
+        else:
```

Ruff (0.15.7)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)